tests/jvm/src/test/resources/metacp.expect

```diff
@@ -121,65 +54,37 @@ _root_.com.javacp.ClassSuffix# => class ClassSuffix.{+2 decls}
   extends Object
 _root_.com.javacp.ClassSuffix#<init>(). => def <init>: (): Unit
   Unit => _root_.scala.Unit#
-_root_.com.javacp.ClassSuffix#suffix. => var suffix: ClassSuffix#Inner[String]#Bar#Fuz[Integer]
-  ClassSuffix#Inner => _root_.com.javacp.ClassSuffix#Inner#
-  String => _root_.java.lang.String#
-  Bar => _root_.Bar#
-  Fuz => _root_.Fuz#
-  Integer => _root_.java.lang.Integer#
-
-com/javacp/Javacp$1.class
--------------------------
-
-Summary:
-Schema => SemanticDB v3
-Uri => com/javacp/Javacp$1.class
-Text => empty
-Language => Java
-Symbols => 4 entries
-
-Symbols:
-_root_.com.javacp.Javacp#1# => private[javacp] class 1.{+2 decls}
+_root_.com.javacp.ClassSuffix#Inner# => private[javacp] class Inner[A <: Object].{+1 decls}
   extends Object
-  extends Serializable
-_root_.com.javacp.Javacp#1#<init>(Javacp). => private[javacp] def <init>: (arg0: Javacp): Unit
-  arg0 => _root_.com.javacp.Javacp#1#<init>(Javacp).(arg0)
+_root_.com.javacp.ClassSuffix#Inner#<init>(ClassSuffix). => private[javacp] def <init>: (arg0: ClassSuffix): Unit
```

Comment:
I like that `this$N` outer fields no longer show up in SemanticDB. Can we also filter out the corresponding outer parameters from inner class constructors? The heuristic would be "if the class contains a field such that `field.name.startsWith("this$")`, then drop the first parameters of the constructor".

Reply:
Done.